docker/BUILD

```diff
@@ -5,6 +5,17 @@ load("//docker:debug_docker.bzl", "debug_docker_build")
 # Use "manual" target tag to skip rules in the wildcard expansion
 
 pkg_tar(
+    name = "agent_tar",
+    extension = "tar.gz",
+    files = [
+        "//cmd/agent",
```

Comment:
istio-agent

For other proxies, we do not want to change all build files and docs.